### PR TITLE
[CDN] upgrade azure-mgmt-cdn to 12.0.0 for Azure Front Door Standard/Premium GA

### PR DESCRIPTION
**Pull request by t-bzhan**
**Description**

Upgrade azure-mgmt-cdn to 12.0.0 for Azure Front Door Standard/Premium GA

**Testing Guide**

azdev test cdn

**History Notes**

---

This checklist is used to make sure that common guidelines for a pull request are followed.

- [x] The PR title and description has followed the guideline in [Submitting Pull Requests](https://github.com/Azure/azure-cli/tree/dev/doc/authoring_command_modules#submitting-pull-requests).

- [x] I adhere to the [Command Guidelines](https://github.com/Azure/azure-cli/blob/dev/doc/command_guidelines.md).

- [x] I adhere to the [Error Handling Guidelines](https://github.com/Azure/azure-cli/blob/dev/doc/error_handling_guidelines.md).

### CODE REVIEW

**Comment by yonzhan** [comment]
CDN

File: src/azure-cli/azure/cli/command_modules/cdn/_help.py [hunk 1]
```diff
@@ -1417,7 +1431,7 @@
   - name: Creates a secret using the specified certificate version.
     text: >
         az afd secret create -g group --profile-name profile --secret-name secret1 --secret-version version1
-        --secret-source /subscriptions/sub1/resourceGroups/rg1/providers/Microsoft.KeyVault/vaults/vault1/certificates/cert1
```
Comment (kairu-ms) [inline src/azure-cli/azure/cli/command_modules/cdn/_help.py:1420]: Can 'certificate' still work or customer has to use 'secrets'?

Comment (t-bzhan) [reply src/azure-cli/azure/cli/command_modules/cdn/_help.py:1420]: Customer could use either certificates or secrets with content-type like "application/x-pem-file" or " "application/x-pkcs12", but RP side only accept Resource Reference with "secret". (I am also confused with that design though).   The certificate could also be accessed using the secret reference id, as specified in: https://docs.microsoft.com/en-us/azure/key-vault/certificates/about-certificates

image

**Review by kairu-ms — APPROVED** [review]

**Comment by RakeshMohanMSFT** [comment]
Fixes https://github.com/Azure/azure-cli/issues/21751

**Pull request merged**